cli/internal/cmd/init.go

```diff
@@ -125,16 +128,14 @@ func (i *initCmd) initialize(cmd *cobra.Command, newDialer func(validator *cloud
 	if err != nil {
 		return err
 	}
-	i.log.Debugf("Got service account uri %s", serviceAccURI)
-	i.log.Debugf("Loading master secret file from %s", flags.masterSecretPath)
```

Comment:
Removed the first one completely since "Successfully" messages are not really needed (if you know the entry point and get to the next message, you know that it passed without an error). Can add it back though.

I removed the master secret log message because that one does not make sense. If the path is "", it will not load any file; it will generate one. readOrGenerateMasterSecret does already plenty of logging in the beginning of both code paths so no information is gone (since masterSecretPath has already been printed earlier two times).

Reply:
Okay, then I'm fine with it as is. :+1: